x/distribution/keeper/keeper_test.go

```diff
@@ -113,10 +111,11 @@ func TestFundCommunityPool(t *testing.T) {
 	app := simapp.Setup(false)
 	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
 
-	addr := simapp.AddTestAddrs(app, ctx, 2, sdk.NewInt(1000000000))
+	addr := simapp.AddTestAddrs(app, ctx, 2, sdk.ZeroInt())
```

Comment:
why changing this test?

Reply:
because this test relied once again on set balance, in this way its more clear that we're creating accounts with no coins and funding them afterwards.